.github/workflows/release.yml

```diff
@@ -191,7 +195,7 @@ jobs:
         env:
           OPENSSL_KEY: ${{ secrets.OPENSSL_KEY }}
         run: |
-          for archive in phylum-*.zip;
+          for archive in phylum-*.zip phylum-*.exe;
```

Comment:
I intentionally left the windows binary unsigned since it is not fully supported and does not contain all of the features of our other builds.

Reply:
It has been reverted.